components/SearchableSelect.vue

```diff
@@ -218,7 +219,7 @@ watchEffect(async () => {
   if (savedQuery === query.value) {
     suggestedOptions.value = options
   }
-})
+}, { debounce: 400, maxWait: 800 })
```

Comment:
Isn't it slightly too much?

Reply:
I found that 300 was firing between 2 keystrokes so I changed it to 400 but I can reduce it back to 300